lighthouse-core/test/audits/byte-efficiency/uses-responsive-images-test.js

```diff
@@ -159,7 +192,7 @@ describe('Page uses responsive images', () => {
       {computedCache: new Map()}
     );
 
-    assert.equal(auditResult.items.length, 2);
+    assert.equal(auditResult.items.length, 3);
```

Comment:
It'd be good to change this assertion to which images are actually failing, otherwise it's hard to determine if the test is working as expected.

Reply:
I've added the specific failures.